### PR TITLE
feat: added new options to the CLI

### PR DESCRIPTION
**Pull request by masterchief164**
Added speakers, categories and tags options to the cli

### CODE REVIEW

**Comment by masterchief164** [comment]
This PR fixes #5

File: app/application.py [hunk 1]
```diff
@@ -67,19 +67,33 @@ def convert(link, model):
...
-
+    tags = tags.strip()
+    tags = tags.split(",")
+    for i in range(len(tags)):
+        tags[i] = tags[i].strip()
+    speakers = speakers.strip()
+    speakers = speakers.split(",")
+    for i in range(len(speakers)):
+        speakers[i] = speakers[i].strip()
+    category = category.strip()
```
Comment (adamjonas) [inline app/application.py:85]: this line crashed on me when I didn't pass in a category

Screen Shot 2023-01-18 at 8 15 01 PM

Comment (adamjonas) [inline app/application.py:85]: This line crashed on me when I didn't pass in a category

Screen Shot 2023-01-18 at 8 18 44 PM

Comment (masterchief164) [reply app/application.py:91]: Fixed the bug category was expecting a string but was getting None.

**Comment by adamjonas** [comment]
I having trouble running through the workflow here. I'm getting some of the errors below:

Screen Shot 2023-01-20 at 9 24 28 PM
Screen Shot 2023-01-20 at 9 24 12 PM

At this point, it makes sense to add some functional tests. This means that you can stub out the downloading and transcription bit and focus on the features and also ensure that new features aren't breaking existing functionality.

**Comment by adamjonas** [comment]
` > tstbtc oubZGyDY4Dc chaincodelabs/chaincodepodcast -t 'Test test test' -s 'Adam Jonas, Murch' -T 'mempool,other' -c 'podcast'` produced `https://github.com/adamjonas/bitcointranscripts/pull/7/files`

The transcript itself is in pretty good shape but the dir and subdir creation with index files needs some cleaning up.

**Comment by adamjonas** [comment]
Unsure if introduced in this PR but the transcripts aren't being cleaned up from a failed run.

See  https://github.com/adamjonas/bitcointranscripts/pull/6,  https://github.com/adamjonas/bitcointranscripts/pull/8,  https://github.com/adamjonas/bitcointranscripts/pull/9

**Comment by masterchief164** [comment]
Shall we move the creation of the test suite to a different issue as we already have a lot of PRs lined up

**Comment by adamjonas** [comment]
This shouldn't be empty, if there is no option that is provided, it shouldn't be in the front matter.

Screen Shot 2023-01-25 at 2 05 24 PM

**Comment by adamjonas** [comment]
I'm going to merge this but https://github.com/bitcointranscripts/tbtbtc/pull/32#issuecomment-1404106196 needs to be addressed

**Comment by masterchief164** [comment]
Should we do this for the other options too (viz. tags, speakers, etc)?

**Comment by adamjonas** [comment]
There should not be empty strings in front matter no matter the type.

**Comment by adamjonas** [comment]
There should not be empty strings in front matter regardless of type.

**Pull request merged**